src/main/scala/tech/cryptonomic/conseil/tezos/TezosOptics.scala

```diff
@@ -105,7 +105,7 @@ object TezosOptics {
       */
     def extractOperationsSequence(
         block: Block
-    ): List[Either[Operation, InternalOperationResults.InternalOperationResult]] =
+    ): List[(Either[Operation, InternalOperationResults.InternalOperationResult], OperationHash)] =
```

Comment:
Same consideration as the parallel PR.

Maybe using the opHash as first element in the tuple and possibly using a wrapping case class could improve the reading experience

Reply:
sure, changed order in the tuple